### PR TITLE
minor updates to the api

### PR DESCRIPTION
**Pull request by demmer**
Minor documentation edits and a change in `connect` so the 'address' option is now 'contactPoints' to match the underlying driver option better.

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -142,17 +142,17 @@ Client.prototype.eachRow = function(query, params, options, rowCallback, endCall
     this._execute(args.query, args.params, args.options, pageCallback, endCallback);
 };
 
-Client.prototype.new_query = function(query, callback) {
-    return this.client.new_query();
+Client.prototype.query = function(query, callback) {
```
Comment (davidvgalbraith) [inline lib/client.js]: It's a little confusing to use `query` as a noun here, don't you think? I would think of `client.query` as telling the client to issue a query, not give me some query object. Also what are the arguments to this function for?

Comment (demmer) [inline lib/client.js]: I agree. I removed the commit that renamed all the constructor methods.

**Comment by demmer** [comment]
@davidvgalbraith updated from a while ago and rebased on top of the latest nan-2 version.

**Comment by davidvgalbraith** [comment]
+1

**Pull request merged**